### PR TITLE
Add test 'Sort and pagination image settings'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test 'Sort and pagination image settings'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/08_design/06_imageSettings/04_sortAndPagination.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20997)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+      const paginationNumber = await imageSettingsPage.selectPaginationLimit(page, '20');
+      expect(paginationNumber).to.equal('1 2');
+    });
+
+    it('should click on next', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnNext', baseContext);
+
+      const paginationNumber = await imageSettingsPage.paginationNext(page);
+      expect(paginationNumber).to.equal('1 2');
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js]: I think here you could check which page is active BY checking text in active item of the pagination list 
image

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    });
+
+    it('should click on previous', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnPrevious', baseContext);
+
+      const paginationNumber = await imageSettingsPage.paginationPrevious(page);
+      expect(paginationNumber).to.equal('1 2');
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js]: Same here

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+  // 3 : Sort image settings
+  describe('Sort image settings table', async () => {
+    const sortTests = [
+      {
+        args: {
+          testIdentifier: 'sortByIdDesc', sortBy: 'id_image_type', sortDirection: 'down', isFloat: true,
+        },
+      },
+      {args: {testIdentifier: 'sortByNameAsc', sortBy: 'name', sortDirection: 'up'}},
+      {args: {testIdentifier: 'sortByNameDesc', sortBy: 'name', sortDirection: 'down'}},
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js]:
```suggestion
      {
        args: {
          testIdentifier: 'sortByNameAsc', sortBy: 'name', sortDirection: 'up',
        },
      },
      {
        args: {
          testIdentifier: 'sortByNameDesc', sortBy: 'name', sortDirection: 'down',
        },
      },
```

Just to have the same thing that others.

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js]: Done thx :+1:

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js]: Done thx :+1:

File: tests/UI/pages/BO/design/imageSettings/index.js [hunk 4]
```diff
@@ -63,6 +67,13 @@ class ImageSettings extends BOBasePage {
     this.bulkActionDropdownMenu = `${this.bulkActionBlock} ul.dropdown-menu`;
     this.selectAllLink = `${this.bulkActionDropdownMenu} li:nth-child(1)`;
     this.bulkDeleteLink = `${this.bulkActionDropdownMenu} li:nth-child(4)`;
+
+    // Pagination selectors
+    this.paginationDropdownButton = `${this.gridForm} .pagination .dropdown-toggle`;
+    this.paginationItems = number => `${this.gridForm} .dropdown-menu a[data-items='${number}']`;
+    this.paginationLabel = `${this.gridForm} .pagination`;
```
Comment (Progi1984) [inline tests/UI/pages/BO/design/imageSettings/index.js]:
```suggestion
    this.paginationLabel = `${this.gridForm} .pagination`;
    this.paginationDropdownButton = `${this.paginationLabel} .dropdown-toggle`;
    this.paginationItems = number => `${this.gridForm} .dropdown-menu a[data-items='${number}']`;
```

**Review by SimonGrn — DISMISSED** [review]

**Review by Progi1984 — COMMENTED** [review]
A small comment

**Review by SimonGrn — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**